### PR TITLE
[FSTORE-425] throw meaningful error if user tries to set time_travel_format=None in a Python kernel

### PR DESCRIPTION
**Pull request by bubriks**
This PR adds/fixes/changes...
- please summarize your changes to the code 
- and make sure to include all changes to user-facing APIs

JIRA Issue: -

Priority for Review: -

Related PRs: -

**How Has This Been Tested?**

- [ ] Unit Tests
- [ ] Integration Tests
- [ ] Manual Tests on VM

**Checklist For The Assigned Reviewer:**

```
- [ ] Checked if merge conflicts with master exist
- [ ] Checked if stylechecks for Java and Python pass
- [ ] Checked if all docstrings were added and/or updated appropriately
- [ ] Ran spellcheck on docstring
- [ ] Checked if guides & concepts need to be updated
- [ ] Checked if naming conventions for parameters and variables were followed
- [ ] Checked if private methods are properly declared and used
- [ ] Checked if hard-to-understand areas of code are commented
- [ ] Checked if tests are effective
- [ ] Built and deployed changes on dev VM and tested manually
- [x] (Checked if all type annotations were added and/or updated appropriately)
```

### CODE REVIEW

**Review by moritzmeister — COMMENTED** [review]
What about the java part?

**Comment by bubriks** [comment]
This pr jira mentioned python in particular so I made changes only for it. As for java, on my current VM spark doesn't work so I am building a new one to test if it works, because I have a suspicion it doesnt.

File: java/src/main/java/com/logicalclocks/hsfs/metadata/FeatureGroupBase.java [hunk 1]
```diff
@@ -102,6 +102,10 @@ public class FeatureGroupBase {
   @Setter
   protected String location;
 
+  @Getter
+  @Setter
+  protected TimeTravelFormat timeTravelFormat = TimeTravelFormat.HUDI;
```
Comment (moritzmeister) [inline java/src/main/java/com/logicalclocks/hsfs/metadata/FeatureGroupBase.java:107]: Make sure for external feature groups the backend doesn't complain if this is being sent.

Comment (bubriks) [reply java/src/main/java/com/logicalclocks/hsfs/metadata/FeatureGroupBase.java:107]: the are no complaints, i can create an external feature group

**Pull request merged**